psa-crypto-sys/build.rs

```diff
@@ -35,6 +35,16 @@
 #![allow(clippy::multiple_crate_versions)]
 
 fn main() -> std::io::Result<()> {
+    // If the prefix feature is not enabled then set the "CARGO_PKG_LINKS"
+    // parameter to mbedcrypto to avoid any duplicate symbols from any other
```

Comment:
Should this be `mbedtls` instead of `mbedcrypto` so that we can detect an attempt to combine with `mbedtls-sys-auto` (https://github.com/fortanix/rust-mbedtls/blob/master/mbedtls-sys/Cargo.toml#L14)?

Reply:
I'm worried changing this would affect if there is another crate that links to `libmbedcrypto`. The user in this case can use the `prefix` feature to avoid any collisions instead of changing `mbedcrypto`?

Reply:
In fact I've also suggested that `mbedtls-sys-auto` change its `links` line: https://github.com/fortanix/rust-mbedtls/issues/289
If both crates changed, then we'd be back to where we started! The `links` line is a very crude mechanism for detecting collisions.

Reply:
Since you have already raised an issue there, I think we can stick to `mbedcrypto` for now.